7.application_configuration.md

```diff
@@ -78,11 +78,17 @@ This section defines the instances of components to create with the application
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `componentName` | `string` | Y | | The name of the component of which to create an instance. |
+| `componentName` | `string` | N | | The name of the component of which to bind application configuration. Using componentName represent the ApplicationConfiguration always track the latest revision. |
```

Comment:
it seems in the implementation, it's `component` not `componentName`?

Reply:
it's componentName, I confirmed